### PR TITLE
Exemptions for containers and namespaces

### PR DESCRIPTION
**Pull request by shaswa**
- Extended ability to add exemptions for containers and Namespaces
- Removed some unused variables in the files touched by this PR
- Updated documentation on how to uae exemptions
- Based on request #408

### CODE REVIEW

File: docs-md/customization/exemptions.md [hunk 1]
```diff
@@ -18,19 +21,25 @@ kubectl annotate deployment my-deployment polaris.fairwinds.com/cpuRequestsMissi
...
       - dns-controller
     rules:
       - hostNetworkSet
-  # exemption valid in all namespaces
+  # exemption valid in all namespaces and dns-controller controller for all containers
   - controllerNames:
       - dns-controller
     rules:
       - hostNetworkSet
+  # exemption valid in kube-system namespace and all controllers for coredns container
```
Comment (baderbuddy) [inline docs-md/customization/exemptions.md:38]: We should probably add one that's a particular namespace but all rules and all controllers and all containers.

**Review by baderbuddy — APPROVED** [review]
This is looking good to me. I just asked for a little more on the docs.

File: docs-md/customization/exemptions.md (hunk 1 above)
Comment (shaswa) [reply docs-md/customization/exemptions.md:38]: Good point! I updated the documentation by borrowing some from #457 PR

**Review by baderbuddy — APPROVED** [review]
LGTM

**Comment by shaswa** [comment]
Thank you, _Bader!

**Pull request merged**